Automattic/newspack-plugin
# feat: newspack plugin settings component

### All Submissions:






### Changes proposed in this Pull Request:

Implements a generic `PluginSettings` component that handles the standardized settings structure implemented by https://github.com/Automattic/newspack-ads/pull/213 and updates the **Placement** wizard page into a more generic **Global settings** page.

### How to test the changes in this Pull Request:

1. Check out this PR and https://github.com/Automattic/newspack-ads/pull/213, then visit Advertising -> Global settings
2. Scroll down and observe the default values displayed
3. Update those values, refresh page
4. Confirm values are updated.

### Other information:
